protos/orchestrator_service.proto

```diff
@@ -57,14 +57,21 @@ message ParentInstanceInfo {
     OrchestrationInstance orchestrationInstance = 4;
 }
 
+message TraceContext {
+    string traceID = 1;
+    string spanID = 2;
+    google.protobuf.StringValue traceState = 3;
```

Comment:
Is the reason that `traceState`'s type is `google.protobuf.StringValue` because it can be `null`, but `traceId` and `spanId` shouldn't? Is there any other reason? Right now, in Durable Task Framework (.NET), we have `spanId` as a nullable `String` so that might be something to change.

Reply:
Yes, exactly. It's a little strange that the SpanId property in the .NET implementation is nullable since a valid span ID should never be null, but I don't remember exactly why it was implemented this way. There might have been other reasons related to how instances of that class get initialized (for example, if they get deserialized from JSON, you can't really guarantee that all properties are non-null - maybe there are other ways we can work around that C#-specific weirdness).